screen_layout

```diff
@@ -0,0 +1,35 @@
+# the following two lines give a two-line status, with the current window highlighted
```

Comment:
Deze zou eigenlijk in het hoofdproject geinclude moeten worden, maar dat is mijn persoonlijke mening...